fili-core/src/main/java/com/yahoo/bard/webservice/web/ResponseData.java

```diff
@@ -88,7 +87,8 @@ public Response(
             SimplifiedIntervalList volatileIntervals,
             Map<String, URI> paginationLinks,
             Pagination pagination,
-            ObjectMappersSuite objectMappers
+            ObjectMappersSuite objectMappers,
+            ApiRequest apiRequest
```

Comment:
We should just remove that function. It no longer belongs on this class, and this is so deep into the guts of Fili, that I don't think we have to worry about removing it breaking anybody.